### PR TITLE
Fixes finnish speak to prevent #triggering admins

### PR DESCRIPTION
**Pull request by Iamgoofball**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 1]
```diff
@@ -69,7 +69,7 @@
 						"SOTP MESING WITH THE ROUNS SHITMAN!!!", \
 						"SKELINGTON IS 4 SHITERS!", \
 						"MOMMSI R THE WURST SCUM!!", \
-						"Oh Fugg :DDDD", \
+						"Oh Fugg colon capital d capital d capital d capital d", \
```
Comment (ihadtoregisterforthis) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: :eggplant:

Comment (Shadowmech88) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: Shouldn't it be "capital dee"?

**Comment by optimumtact** [comment]
STOP THE SINFUL HAND

**Comment by 9600bauds** [comment]
xddd

**Comment by carshalash** [comment]
I really made a monster.

**Comment by Shadowmech88** [comment]
Was it worth it?

**Comment by carshalash** [comment]
Not at all.

**Comment by AutismSpider** [comment]
job well done put it in

**Comment by 9600bauds** [comment]
Just ten lines above this finnishgarble, you can see a brain damage line that says ";_;"
And above that, greentexting IC.

**Comment by AutismSpider** [comment]
@9600bauds yup but the finnish meme is more deadly then that feel when and ton of other shit

**Comment by Gringonius** [comment]
At this point and with the train of thought people are following I think that the majority of retard lines need to be changed as they're mostly ancient and forgotten /tg/-centric memes.

**Comment by clusterfack** [comment]
Too late its dead DEAD

**Pull request closed without merging**